### PR TITLE
Perf optimization

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: amp-camp/src/html/pages/index.html [hunk 1]
```diff
@@ -14,10 +14,10 @@
     <main id="content" role="main" class="landing">
         <section class="hero-wrapper">
             <a href="/product-listing?gender=women&category=shirts" class="block">
-                <amp-img src="img/women-narrow.jpg" width="1.25" height="1" layout="responsive" media="(max-width: 52.06rem)" alt="Hero Landing 1" noloading="">
```
Comment (morsssss) [inline amp-camp/src/html/pages/index.html]: we inherited these from the template. I assume you simply want to formalize these with HTML classes?

File: amp-camp/src/html/partials/menu.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
         <ul class="label">
             <li class="logo">
                 <a href="/" class="block">
-          <amp-img src="img/logo-nav.png" width="180" height="73" layout="intrinsic" alt="Company logo" noloading="">
+          <amp-img src="img/logo-nav.png" width="180" height="73" layout="responsive" style="max-width: 180px" alt="Company logo" noloading="">
```
Comment (morsssss) [inline amp-camp/src/html/partials/menu.html]: I was curious about the motivation for this change!

**Comment by sebastianbenz** [comment]
The explanation was hidden in the "Improve FCP by 50%" commit message:

> The AMP boilerplate delays rendering until the AMP runtime is loaded.
AMP Optimizer can remove the boilerplate, but can't do this (yet) if
sizes or media attributes are used on AMP elements.
> 
> This commit replaces these with plain CSS to enable removing the
boilerplate.

**Comment by morsssss** [comment]
That's really interesting - about why we need to remove the size/media attributes.

I want to start providing lots of comments in our HTML. Would you mind starting an area for that on this page? Maybe toward the top. This would be a great comment to include there.

**Comment by sebastianbenz** [comment]
@morsssss I'm torn about whether it makes sense to document this right now. Support for intrinsic and media are planned for optimizer so it'd be only temporary.

**Comment by morsssss** [comment]
Got it. Never mind then!

**Comment by morsssss** [comment]
Please merge away 😄

**Comment by sebastianbenz** [comment]
Thanks Ben!

**Pull request merged**